backend/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM node:15.0.1
+WORKDIR /foodies/backend
+COPY *.json /foodies/backend/
+RUN npm install
+# build
+COPY . .
+RUN npm run build
+RUN npm prestart
```

```suggestion
RUN yarn run build
RUN yarn run prestart
```

Just changing these commands to use `yarn` instead of `npm` so we're not dealing with two `.lock` files (which could lead to inconsistencies)